teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(name = 'Unknown', specialization:, age: nil, parent_permission: true)
```

Comment:
- [ ] I still got the same error that the previous reviewer detected, kindly ensure that the **`specialization`** variable has a value attached to it or you remove the **`:`** notation. ℹ️ 

<img width="573" alt="Untitled" src="https://github.com/ClintonYounge/School-Libraby-Project/assets/103386183/5c7a0bae-3351-461d-bcb8-cdbc4316a8b1">

```suggestion
  def initialize(name = 'Unknown', specialization, age: nil, parent_permission: true)
```